basiclti/basiclti-tool/pom.xml

```diff
@@ -73,6 +76,7 @@
         <dependency>
             <groupId>com.nimbusds</groupId>
             <artifactId>nimbus-jose-jwt</artifactId>
+            <scope>provided</scope>
```

Comment:
Because this dependency was added to master there is no need to override the scope
```suggestion
```